_install-and-configure/plugins.md

```diff
@@ -85,13 +85,13 @@ opensearch-node1 opensearch-notifications-core        2.0.1.0
 
 There are three ways to install plugins using the `opensearch-plugin`:
 
-- [Install a plugin by name]({{site.url}}{{site.baseurl}}/opensearch/install/plugins#install-a-plugin-by-name)
-- [Install a plugin by from a zip file]({{site.url}}{{site.baseurl}}/opensearch/install/plugins#install-a-plugin-from-a-zip-file)
-- [Install a plugin using Maven coordinates]({{site.url}}{{site.baseurl}}/opensearch/install/plugins#install-a-plugin-using-maven-coordinates)
+- [Install a plugin by name](#install-a-plugin-by-name)
+- [Install a plugin by from a zip file](#install-a-plugin-from-a-zip-file)
+- [Install a plugin using Maven coordinates](#install-a-plugin-using-maven-coordinates)
 
 ### Install a plugin by name:
 
-For a list of plugins that can be installed by name, see [Additional plugins]({{site.url}}{{site.baseurl}}/opensearch/install/plugins#additional-plugins).
+For a list of plugins that can be installed by name, see [Additional plugins](#additional-plugins).
 
 #### Usage:
```

Comment:
Global: Please remove terminal punctuation from headings.

Reply:
I am going to take care of it in `_install-and-configure/plugins.md` document.
If heading creates an URL link with `#page_anchors` then punctuation does not feel right (aside from other conventions).

Reply:
Thank you for doing this, @lukas-vlcek! I know this was not part of your changes and we appreciate you going an extra mile!

Reply:
@kolchfa-aws I added these changes to my second commit (that was fixing some broken links in plugins page) so that they can be cherry-picked easily.